packages/flutter_tools/lib/src/drive/web_driver_service.dart

```diff
@@ -247,102 +273,80 @@ Map<String, dynamic> getDesiredCapabilities(
   bool? headless, {
   List<String> webBrowserFlags = const <String>[],
   String? chromeBinary,
-}) {
-  switch (browser) {
-    case Browser.chrome:
-      return <String, dynamic>{
-        'acceptInsecureCerts': true,
-        'browserName': 'chrome',
-        'goog:loggingPrefs': <String, String>{
-          async_io.LogType.browser: 'INFO',
-          async_io.LogType.performance: 'ALL',
-        },
-        'goog:chromeOptions': <String, dynamic>{
-          if (chromeBinary != null)
-            'binary': chromeBinary,
-          'w3c': true,
-          'args': <String>[
-            '--bwsi',
-            '--disable-background-timer-throttling',
-            '--disable-default-apps',
-            '--disable-extensions',
-            '--disable-popup-blocking',
-            '--disable-translate',
-            '--no-default-browser-check',
-            '--no-sandbox',
-            '--no-first-run',
-            if (headless!) '--headless',
-            ...webBrowserFlags,
-          ],
-          'perfLoggingPrefs': <String, String>{
-            'traceCategories':
-            'devtools.timeline,'
-            'v8,blink.console,benchmark,blink,'
-            'blink.user_timing',
+}) =>
+    switch (browser) {
+      Browser.chrome => <String, dynamic>{
+          'acceptInsecureCerts': true,
```

Comment:
Arrow syntax is being abused here (at least at the top level)

https://dart.dev/guides/language/effective-dart/usage#consider-using--for-simple-members